politeiawww/api/www/v1/v1.go

```diff
@@ -56,6 +56,7 @@ const (
 	RouteAllVoteStatus            = "/proposals/votestatus"
 	RouteProposalPaywallDetails   = "/proposals/paywall"
 	RouteProposalPaywallPayment   = "/proposals/paywallpayment"
+	RouteShortProposalDetails     = "/proposals/{tokenPrefix:[A-z0-9]{6}}"
```

Comment:
Not sure what the convention is for path params, but camel case looks weird to me in a uri.  Would prefer just `tokenprefix`.